R/Widget_BarChart.R

```diff
@@ -41,6 +41,16 @@ Widget_BarChart <- function(
   strShinyGroupSelectID = "GroupID",
   bDebug = FALSE
 ) {
+  stopifnot(
```

Comment:
That said I just followed the convention of the `stopifnot()` chunk that was in the `Analyze_xxx()` functions